cocoa/src/toga_cocoa/widgets/activityindicator.py

```diff
@@ -16,7 +16,6 @@ def create(self):
 
         # Add the layout constraints
         self.add_constraints()
-        self.rehint()
```

Comment:
`rehint` calls in `create` methods are redundant, because there's already a call in `container.setter`.